zkfacade/src/test/java/com/yahoo/vespa/curator/stats/LockInfoSamplesTest.java

```diff
@@ -0,0 +1,58 @@
+package com.yahoo.vespa.curator.stats;// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
Bug in test template? Copyright should be on first line.

Reply:
Bug in IntelliJ/have never been able to figure it out.  Fixed.